src/commands/ls.js

```diff
@@ -160,18 +196,18 @@ const ls = {
         if (options.t) {
           // Sort by date modified.
           return ((a.data.mtime < b.data.mtime) ? 1 :
-            (b.data.mtime < a.data.mtime) ? -1 :
-            0);
+              (b.data.mtime < a.data.mtime) ? -1 :
+                  0);
         }
         // Sort alphabetically - default.
         const aFileName = fileFromPath(a.file)
-          .trim()
-          .toLowerCase()
-          .replace(/\W/g, '');
+            .trim()
```

Comment:
`.editorconfig` may be helpful when working on different platforms or with many people.